src/main/java/com/adivinaLaPalabra/squad2/back/AdivinaLaPalabra/exceptions/ErrorHandler.java

```diff
@@ -43,6 +44,14 @@ private ErrorResponseDTO handleNotFoundException(NoHandlerFoundException e) {
         return new ErrorResponseDTO(HttpStatus.NOT_FOUND.value(), HttpStatus.NOT_FOUND.getReasonPhrase());
     }
 
+    @ExceptionHandler({ RangeException.class })
```

Comment:
En vez de meter una entrada para cada nueva excepción, lo que podéis hacer es crear una jerarquía. Vuestra BadRequestException puede ser padre de RangeException y de otras excepciones similares. Otra posibilidad es tener un handler único para este tipo de excepciones y tener un Map que relacione el tipo de excepción con su mensaje de error, status, etc. Dadle una vuelta a ver qué se puede hacer para simplificar el código.